[2025-02-16T16:13:05.722+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-16T16:13:05.748+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_apod_pipeline.export_to_postgres manual__2025-02-16T16:12:58.632286+00:00 [queued]>
[2025-02-16T16:13:05.758+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_apod_pipeline.export_to_postgres manual__2025-02-16T16:12:58.632286+00:00 [queued]>
[2025-02-16T16:13:05.759+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-02-16T16:13:05.792+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): export_to_postgres> on 2025-02-16 16:12:58.632286+00:00
[2025-02-16T16:13:05.801+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1834) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-16T16:13:05.803+0000] {standard_task_runner.py:63} INFO - Started process 1838 to run task
[2025-02-16T16:13:05.803+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nasa_apod_pipeline', 'export_to_postgres', 'manual__2025-02-16T16:12:58.632286+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/nasa_pipeline_dag.py', '--cfg-path', '/tmp/tmpe762nrk0']
[2025-02-16T16:13:05.806+0000] {standard_task_runner.py:91} INFO - Job 66: Subtask export_to_postgres
[2025-02-16T16:13:05.867+0000] {task_command.py:426} INFO - Running <TaskInstance: nasa_apod_pipeline.export_to_postgres manual__2025-02-16T16:12:58.632286+00:00 [running]> on host dfa5576bf540
[2025-02-16T16:13:06.022+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nasa_apod_pipeline' AIRFLOW_CTX_TASK_ID='export_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-02-16T16:12:58.632286+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-16T16:12:58.632286+00:00'
[2025-02-16T16:13:06.025+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-16T16:13:06.069+0000] {base.py:84} INFO - Using connection ID 'postgres_nasa' for task execution.
[2025-02-16T16:13:06.095+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-16T16:13:06.095+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-16T16:13:06.119+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nasa_apod_pipeline, task_id=export_to_postgres, run_id=manual__2025-02-16T16:12:58.632286+00:00, execution_date=20250216T161258, start_date=20250216T161305, end_date=20250216T161306
[2025-02-16T16:13:06.178+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-16T16:13:06.206+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-16T16:13:06.315+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
